examples/src/main/python/transitive_closure.py

```diff
@@ -23,7 +23,6 @@
 from pyspark import SparkContext
 
 numEdges = 200
-numVertices = 100
```

Comment:
Wait, actually isn't the bug below? `rand.randrange(0, numEdges)` should be `rand.randrange(0, numVertices)`